bindings_ffi/Cargo.toml

```diff
@@ -11,14 +11,19 @@ crate-type = ["lib", "cdylib"]
 [dependencies]
 log = { version = "0.4", features = ["std"] }
 thiserror = "1.0.40"
-uniffi = { git = "https://github.com/mozilla/uniffi-rs", rev = "6938c262293802f00a373f0d1aa16a2079071b24", features = ["tokio", "cli"] }
-uniffi_macros = { git = "https://github.com/mozilla/uniffi-rs", rev = "6938c262293802f00a373f0d1aa16a2079071b24" }
+uniffi = { git = "https://github.com/mozilla/uniffi-rs", rev = "cae8edc45ba5b56bfcbf35b60c1ab6a97d1bf9da", features = [
+    "tokio",
+    "cli",
+] }
+uniffi_macros = { git = "https://github.com/mozilla/uniffi-rs", rev = "cae8edc45ba5b56bfcbf35b60c1ab6a97d1bf9da" }
```

Comment:
Do we want to stick with hardcoded commits or should we switch to using tags? They would at least communicate more clearly which version we are using. I realize that tags can move, but we presumably assume responsible maintainer regardless and the fact that it can move would be a feature, not a bug.

Reply:
In general I'd be in favor of this -- though it does look like the last [release](https://github.com/mozilla/uniffi-rs/releases/tag/v0.24.3) doesn't include the given change. 

 > the fact that it can move would be a feature, not a bug.
 
This is not always true in rust, particularly when it comes to package version resolution. Changing of a dependency version can cause conflicts in other packages if they have strict dependency requirements.